test/blueprints/Do.Test.Blueprints.Service.Test/DataAccess/PersistingEntity.cs

```diff
@@ -0,0 +1,61 @@
+﻿namespace Do.Test.DataAccess;
+
+public class PersistingEntity : TestServiceSpec
+{
+    [Test]
+    public void Created_entity_persists()
+    {
+        var entity = GiveMe.AnEntity();
+
+        entity.ShouldBeInserted();
+    }
+
+    [Test]
+    public void Entity_does_not_persist_when_deleted()
+    {
+        var entity = GiveMe.AnEntity();
+
+        entity.ShouldBeInserted();
+
+        entity.Delete();
+
+        entity.ShouldBeDeleted();
+    }
+
+    [Test]
+    public async Task Entity_update_commits_when_done_asynchronously_if_an_error_occurs()
```

Comment:
i don't understand this case. your case tests this behaviour only by chance. let's discuss this

Reply:
besides that, this case is not about persisting, this is about transaction